content/en/mobile/_index.md

```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
 
 **Note:** To set up or edit a dashboard, you need to [login to the Datadog browser app][9]. For more information, see [Dashboards][5]. 
 
-<!--- ## Quick Actions  --->
+## Quick Actions
 
-<!--- {{< img src="mobile/quick_actions.png" alt="dashboard page">}}  --->
+{{< img src="mobile/quick-actions-3.png" alt="Quick Actions">}}
```

Comment:
Changed image. in doc https://docs.google.com/document/d/1qgdHEfvBjmXNsCyLc2K4Gejl5-8wllUxhJMFhVR38rc/edit